src/Stoa.ts

```diff
@@ -21,11 +22,64 @@ class Stoa {
             res.send(req.params.address + ':' + req.query.height);
         });
 
+        /**
+         * When a request is received through the `/push` handler
+         * JSON block data is parsed and stored on each storage.
+         */
         this.stoa.post("/push",
             (req: express.Request, res: express.Response, next: express.NextFunction) => {
-            console.log(req.body);
-            res.status(200).send();
-        });
+            var data: any = req.body;
+            var block: any;
+
+            if (data.block == undefined)
+            {
+                res.status(400).send("Block data is not defined");
+                return;
+            }
+
+            try {
+                block = JSON.parse(data.block);
```

Comment:
If you access `req.body.block`... Isn't it already parsed ?

Reply:
No, it's String.
It should be sent to application/json.